opdracht4.js

```diff
@@ -0,0 +1,138 @@
+// #### Opdracht 4 -Functies
+//
+// Maak deze gehele opdracht eerst alsof je het voor één tv doet. We gaan één tv weergeven in het volgende format:
+//
+//     ```
+//   Philips 43PUS6504/12 - 4K TV
+//   €379,-
+//   43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+//   ```
+//
+//     * **Opdracht 4a:** Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft in het format `[merk] [type] - [naam]` zoals `Philips 43PUS6504/12 - 4K TV` of `NIKKEI NH3216SMART - HD smart TV`.
+//
+
+//1. functie schrijven die merk + type + naam returned, en buiten de scope de variabel waarin het is opgeslagen returned.
+
+function tvFullName () {
+    const fullName = inventory.map((inventory) => {
+        return inventory.brand + " " + inventory.type + " " + inventory.name;
```

Comment:
Qua leesbaarheid vind ik dit altijd iets makkelijker
```suggestion
        return `${inventory.brand} ${inventory.type} ${inventory.name}`;
```